### PR TITLE
plugins:foreman_docker - Release 1.0.1

### PR DESCRIPTION
**Pull request by eLobato**
https://rubygems.org/gems/foreman_docker/versions/1.0.1

### CODE REVIEW

**Comment by mmoll** [comment]
scratchbuild? :)

File: plugins/ruby-foreman-docker/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-docker (1.0.1-0) stable; urgency=low
```
Comment (domcleal) [inline plugins/ruby-foreman-docker/debian/changelog]: -0 should be -1 or missing?

**Comment by domcleal** [comment]
http://ci.theforeman.org/job/packaging_build_deb_plugin/386/ running.

**Comment by domcleal** [comment]
Thanks @eLobato, building for 1.7 and develop.

**Pull request merged**